app/assets/javascripts/plate-map/overlay.js

```diff
@@ -14,41 +14,42 @@ var plateLayOutWidget = plateLayOutWidget || {};
         this.overLayTextContainer.text("Completion Percentage:");
         this.overLayContainer.append(this.overLayTextContainer);
         this.overLayButtonContainer = this._createElement("<div></div>").addClass("plate-setup-overlay-button-container");
+        this.overLayButtonContainer.attr("id", "plate-map-control-button-container-id");
```

Comment:
Use a class, not id

Reply:
Don't need id at all here, should never be referencing this by id in usual code.